content/release-notes/v1.0.264.mdx

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: 1.0.264
+date: 2023-02-09
+---
+
+Released on Thursday February 9th, 2023.
+
+- Child Products (https://ordercloud.io/knowledge-base/child-products)
```

Comment:
you'll want to format links like this:
`<Link to="//knowledge-base/child-products">Child Products</Link>.`